includes/utils.php

```diff
@@ -656,4 +656,31 @@ public static function is_empty( $source, $key = false ) {
 	public static function has_pro() {
 		return defined( 'ELEMENTOR_PRO_VERSION' );
 	}
+
+	/**
+	 * Convert HTMLEntities to UTF-8 characters,
+	 * then encode the title to escape problematic characters in URLs
+	 *
+	 * @param $string
+	 * @return string
+	 */
+	public static function urlencode_htmlentities( $string ) {
```

Comment:
Do only `html_entity_decode` functionality

Reply:
Why? now some titles will throw error at sharing.

Reply:
In order to keep the generic of this method, it will be used like this:

`rawurlencode( Utils::decode_htmlentities( YOUR_TEXT ) )`

Reply:
But the only use case for this util is for encode URL's properly, so the user shouldn't think about `rawurlencode`.

The method name may be `Utils::encode_url`;

Reply:
I agree with you on this. Done.